### PR TITLE
list user following

### PR DESCRIPTION
**Pull request by nguyenttngan**
list all users that they are following the authenticated user

### CODE REVIEW

File: app/Http/Controllers/Web/FollowsController.php [hunk 1]
```diff
@@ -15,8 +15,20 @@ class FollowsController extends Controller
      */
     public function follow(User $user)
     {
-        Auth::user()->follows()->toggle($user->id);
+            Auth::user()->follows()->toggle($user->id);
```
Comment (tuanpht) [inline app/Http/Controllers/Web/FollowsController.php]: 8 spaces?

Comment (nguyenttngan) [inline app/Http/Controllers/Web/FollowsController.php]: vâng em sửa ngay ạ

**Comment by tuanpht** [comment]
ok

File: resources/views/web/follows/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        @foreach($users as $user)
+            <li class="list-group-item col-md-6 col-md-offset-3">
+                <a href="{{ action('Web\UsersController@show', ['user' => $user->id]) }}">
+                    <img src="{{ $user->avatarUrl }}" alt="avatar" class="avatar-navigation"> {{ $user->name }}</a></li>
```
Comment (Framgia-HoVanTuan) [inline resources/views/web/follows/show.blade.php]: Convention.
cho `</a></li>` xuống mỗi thẻ 1 dòng.

Comment (nguyenttngan) [inline resources/views/web/follows/show.blade.php]: vâng ạ

**Comment by dung13890** [comment]
OK

**Comment by Framgia-HoVanTuan** [comment]
Ok.

**Pull request merged**